PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -435,6 +435,19 @@ extension PocketSource {
     }
 }
 
+// MARK: - Search
+extension PocketSource {
+    public func search(for term: String, scope: SearchScope) {
+        let operation = operations.searchList(
+            apollo: apollo,
+            scope: scope,
+            term: term,
+            events: _events
+        )
+        enqueue(operation: operation, task: .searchList(term: term, scope: scope))
```

Comment:
I'm wondering if it makes sense to enqueue these operations as we do other actions, such as favoriting. Search almost feels like it could be its own service, where its search can run in parallel to this queue. My thoughts are that if a user's offline and stacks a _bunch_ of item actions (archive, favorites, etc), opens search, it's enqueued, then goes online, then the search technically runs, no? Based on this implementation.

Reply:
Yeah, I saw a delay in search too while waiting for the fetch list to load, so considering the conversation regarding `Async`, I'll remove this. In terms of the user being offline and enqueuing, I'll make those considerations when working on this ticket `https://getpocket.atlassian.net/browse/IN-980`.

Reply:
Discussed over zoom and decided to create a separate service without using OperationQueues to avoid complexity